COVIDWatch.xcodeproj/project.pbxproj

```diff
@@ -655,6 +656,24 @@
 			shellScript = "\"${PODS_ROOT}/Target Support Files/Pods-COVIDWatch iOS/Pods-COVIDWatch iOS-frameworks.sh\"\n";
 			showEnvVarsInLog = 0;
 		};
+		E206411D243EB03A00FB08AE /* SwiftLint */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			name = SwiftLint;
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "if which swiftlint >/dev/null; then\n  swiftlint\nelse\n  echo \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\nfi\n";
```

Comment:
Does this find the SwiftLint in the `Pods` folder?  Or does it find one that you have installed globally?  I wonder if this should be explicit to use the `Pods` one by using `${PODS_ROOT}/SwiftLint/swiftlint` and remove the which.  We know its there so we probably don't need the warning.